### PR TITLE
Contacts 2.0.0

### LINKED ISSUE #144: No export button on NC10
The export button in the settings is missing. The html code is there, if i remove the class 'ng-hide', the button apears and works.

I suspect https://github.com/nextcloud/contacts/blob/master/js/components/addressBook/addressBook_controller.js#L25. But need to look closer.

I updated to version 1.5.3 and have Nextcloud version":"9.1.4.1"

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/42397426-no-export-button-on-nc10?utm_campaign=plugin&utm_content=tracker%2F46751899&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F46751899&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by skjnldsv**
- Remove export version comparison
- Bump min nextcloud version
- Changelog update

Fix #144 

Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -29,8 +29,8 @@
 	<repository type="git">https://github.com/nextcloud/contacts.git</repository>
 	<screenshot>https://raw.githubusercontent.com/nextcloud/screenshots/master/apps/Contacts/contacts.png</screenshot>
 	<dependencies>
-		<owncloud min-version="9.0" max-version="9.2" />
-		<nextcloud min-version="9" max-version="13" />
+		<owncloud min-version="9.2" />
```
Comment (MariusBluem) [inline appinfo/info.xml]: ownCloud tag not needed for Nextcloud 11 🙈

File: appinfo/info.xml [hunk 2]
```diff
@@ -29,8 +29,8 @@
...
+		<nextcloud min-version="11" max-version="13" />
 	</dependencies>
 	<ocsid>168708</ocsid>
```
Comment (MariusBluem) [inline appinfo/info.xml]: ocsid not needed for new AppStore where apps for Nextcloud 11+ are published 😇

Comment (skjnldsv) [inline appinfo/info.xml]: @MariusBluem Do you have a doc where all of those are written? I tried to find it but the doc don't seemed to be up-to-date :'(

Comment (irgendwie) [inline appinfo/info.xml]: @skjnldsv https://nextcloudappstore.readthedocs.io/en/latest/developer.html#app-metadata

**Comment by skjnldsv** [comment]
Sooooo, we're good? :D

**Comment by irgendwie** [comment]
Let **me** merge this shortly before release!

**Comment by skjnldsv** [comment]
> xh3n1 approved these changes 27 minutes ago
> jancborchardt approved these changes 22 minutes ago View changes 
> jonatoni approved these changes 33 seconds ago View changes

Feel the pressure @irgendwie 😆

**Comment by jonatoni** [comment]
hahahahaha @irgendwie take your time 😛

**Comment by skjnldsv** [comment]
Please wait for a quick fix on master.

**Comment by skjnldsv** [comment]
Wait for #332 pls

**Comment by MorrisJobke** [comment]
> Wait for #332 pls

It is in 😉

**Comment by RainerEmrich** [comment]
Any chance that you release any time soon? I don't see what's still blocking the release.

**Comment by skjnldsv** [comment]
@RainerEmrich we're on feature freeze. This will be published very soon. We're testing the release to ensure that no bugs were introduced. Feel free to help us test it with the latest nightlies on the homepage. :)

**Comment by RainerEmrich** [comment]
Ok, I gave it a try and found a bug immediately #364 . That's really unfortunate, because I need this feature.
Changing back to 1.5.3 restores the sorting functionality.

**Comment by skjnldsv** [comment]
@RainerEmrich thank you! We'll handle this asap! :)

**Pull request merged**